data-prepper-core/src/test/java/org/opensearch/dataprepper/parser/model/PipelineConfigurationTests.java

```diff
@@ -155,108 +110,102 @@ public void testOnlySourceAndSink() {
         assertThat(pipelineConfiguration.getReadBatchDelay(), CoreMatchers.is(TestDataProvider.DEFAULT_READ_BATCH_DELAY));
     }
 
-    @Test //not using expected to assert the message
-    public void testNoSourceConfiguration() {
-        try {
-            new PipelineConfiguration(
-                    null,
-                    TestDataProvider.validSingleConfiguration(),
-                    TestDataProvider.validMultipleConfiguration(),
-                    TestDataProvider.validMultipleConfiguration(),
-                    TestDataProvider.TEST_WORKERS, TestDataProvider.TEST_DELAY);
-        } catch (IllegalArgumentException ex) {
-            assertThat(ex.getMessage(), is("Invalid configuration, source is a required component"));
-        }
+    @Test
+    void testNoSourceConfiguration() {
+        final PipelineModel pipelineModel = mock(PipelineModel.class);
```

Comment:
Don't we want to pass `null` for source here?

Reply:
Mockito's default behavior is to provide `null` when not mocked (it is different for collections and primitives).